testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/nativeauth/SignUpAttributesFragment.kt

```diff
@@ -80,23 +79,21 @@ class SignUpAttributesFragment : Fragment() {
             try {
                 val attributes = UserAttributes.Builder
 
-//                val attr1Key = binding.attr1KeyText.text.toString()
-//                if (attr1Key.isNotBlank()) {
-//                    val attr1Value = binding.attr1ValueText.toString()
-//                    attributes
-//                        .customAttribute(attr1Key, attr1Value)
-//                }
-//
-//                val attr2Key = binding.attr2KeyText.text.toString()
-//                if (attr2Key.isNotBlank()) {
-//                    val attr2Value = binding.attr2ValueText.toString()
-//                    attributes
-//                        .customAttribute(attr2Key, attr2Value)
-//                }
+                val attr1Key = binding.attr1KeyText.text.toString()
+                if (attr1Key.isNotBlank()) {
```

Comment:
Let's not do this check. We may want to execute tests that validate whether the SDK's string check works correctly, or whether the API's string check works correctly. Same for the one below.